pkg/cli/zip_cluster_wide.go

```diff
@@ -108,11 +112,47 @@ var debugZipTablesPerCluster = []string{
 	"crdb_internal.table_indexes",
 }
 
+// Tables containing cluster-wide info that are collected using SQL
```

Comment:
Is it possible to have a base set that's shared? I think it'll be valuable to easily determine what's different between the two.